src/test/java/ch/jalu/configme/properties/StringPropertyTest.java

```diff
@@ -92,4 +108,92 @@ void shouldReturnStringFromBoolean() {
         // then
         assertThat(value, equalTo("false"));
     }
+
+    @Test
+    void shouldReadMultipleLines() throws IOException {
```

Comment:
These are great tests! I think they don't belong to StringPropertyTest though because they don't test code inside  `StringProperty`. I think a simple test (without reading/writing files) would be good in SnakeYamlNodeBuilderImplTest to cover the new scalar type as a conventional unit test. As for these two tests here, as mentioned, I absolutely love them, I would just suggest to move them maybe in their own file. I see we have a lot of `YamlFileResource...Test` classes, which is how I've been keeping "integration tests" so far.